src/app/shared/explore/section-component/facet-focus-section/facet-focus-section.component.html

```diff
@@ -0,0 +1,18 @@
+<div *ngIf="(targetFacet$ | async) != null">
+    <div class="card border-0">
+        <h4 class="card-header bg-primary text-white">{{ 'explore.facet-focus-section.' + targetFacet.name + '.title' | translate }}</h4>
+        <div class="border border-gray-100">
+            <div *ngFor="let facetValue of targetFacet._embedded.values; let last = last"
+                class="facet-value d-flex justify-content-between p-3 w-100"
+                [class.border-bottom]="!last"
+                (click)="navigateToFacetView(targetFacet, facetValue)">
+                <p class="text-primary m-0">
```

Comment:
`p` seems not a good idea here --> default display for `p` is `block`. `span` seems a better idea